src/index.css

```diff
@@ -11,3 +11,69 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
```

Comment:
These CSS changes should be moved to `src/Components/WeatherSounds/WeatherSounds.css` and imported into the component.